Dockerfile.metering-ansible-operator

```diff
@@ -47,13 +47,9 @@ COPY charts/openshift-metering ${HELM_CHART_PATH}
 
 COPY manifests/deploy/openshift/olm/bundle /manifests
 
-# to allow running as non-root
-RUN chown -R 1001:0 $HOME && \
-    chmod -R 774 $HOME /etc/passwd
-
 USER 1001
 
-ENTRYPOINT ["/opt/ansible/scripts/entrypoint.sh"]
+ENTRYPOINT ["tini", "--", "/usr/local/bin/ansible-operator", "run", "ansible", "--watches-file", "/opt/ansible/watches.yaml"]
```

Comment:
```suggestion
ENTRYPOINT ["tini", "--", "/usr/local/bin/ansible-operator", "exec-entrypoint", "ansible", "--watches-file", "/opt/ansible/watches.yaml"]
```

Reply:
With this Dockerfile, we pull from the ansible-operator's origin images, which uses a custom operator-sdk version where the `exec-entrypoint` is not a valid sub-command:

```bash
tflannag@localhost operator-metering [update-entrypoint-script] ▶ docker run -it  --network host --entrypoint=/bin/bash quay.io/openshift/origin-ansible-operator:latest
bash-4.2$ /usr/local/bin/ 
ansible-operator  ao-logs           entrypoint        
bash-4.2$ /usr/local/bin/ansible-operator version
operator-sdk version: "dev", commit: "b19ca7bad19dd5c64cadc94ebf7d8ba21a2ec9d8", go version: "go1.13.4 linux/amd64"
bash-4.2$ 
```

When testing this locally, there's no registered `exec-entrypoint` sub-command:
```bash
tflannag@localhost operator-metering [update-rhel-metering-ansible-operator-dockerfile] ▶ docker run -it --network host --entrypoint=/bin/bash quay.io/tflannag/origin-metering-ansible-operator:passwd
bash-4.2$ /usr/local/bin/ansible-operator --help
An SDK for building operators with ease

Usage:
  operator-sdk [command]

Available Commands:
  add         Adds a controller or resource to the project
  alpha       Run an alpha subcommand
  build       Compiles code and builds artifacts
  completion  Generators for shell completions
  generate    Invokes specific generator
  help        Help about any command
  migrate     Adds source code to an operator
  new         Creates a new operator application
  olm-catalog Invokes a olm-catalog command
  print-deps  Print Golang packages and versions required to run the operator
  run         Runs a generic operator
  scorecard   Run scorecard tests
  test        Tests the operator
  up          Launches the operator
  version     Prints the version of operator-sdk

Flags:
  -h, --help      help for operator-sdk
      --verbose   Enable verbose logging

Use "operator-sdk [command] --help" for more information about a command.
bash-4.2$ /usr/local/bin/ansible-operator       
```

Reply:
👍  We were hitting a similar problem. This PR helped us a bunch, thought maybe it was just a miss.

Reply:
No worries, ping me if you need any help with getting your ansible-based Operator up-and-running.